app/views/account/admin/billing_contracts/index.html.slim

```diff
@@ -1,10 +1,15 @@
 .container-wrapper
   .new-button
-    = link_to 'New billing contract',
+    = link_to t('.btn-new'),
       new_account_admin_billing_contract_path,
       class: 'btn btn-primary'
-  h1
-    | All billing contracts
+  h1 
+    = t('.title') 
   .row
+    .col-md-12
     - @billing_contracts.each do |billing_contract|
       = render billing_contract
+  .row
+    .col-md-12
+      .pagination-container
+        = paginate @billing_contracts, theme: 'twitter-bootstrap-4', pagination_class: "pagination-sm"
```

Comment:
use pagination_class: 'pagination-md', in my humble opinion 'sm' is too small, and that size is not user-friendly.
Also, if it possible, use one type of quotes in one file.